changelog/v0.3.0/remove-helm-2-hook.yaml

```diff
@@ -0,0 +1,4 @@
+changelog:
+- type: BREAKING_CHANGE
```

Comment:
i think we can just mark this as a FIX or non-user-facing as this feature isn't in use anywhere

Reply:
Marked as non-user-facing since no Helm 2 clients can currently be consuming these charts anyway.